packages/actor-query-operation-filter-sparqlee/test/ActorQueryOperationFilterSparqlee-test.ts

```diff
@@ -148,6 +149,22 @@ describe('ActorQueryOperationFilterSparqlee', () => {
       await new Promise<void>(resolve => output.bindingsStream.on('error', () => resolve()));
     });
 
+    it('Should log warning for an expressionError', async() => {
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      const logWarnSpy = jest.spyOn(<any> actor, 'logWarn');
+      // eslint-disable-next-line no-import-assign
+      Object.defineProperty(sparqlee, 'isExpressionError', { writable: true });
+      (<any> sparqlee).isExpressionError = jest.fn(() => true);
+      const op = { operation: { type: 'filter', input: {}, expression: erroringExpression }};
+      const output: IActorQueryOperationOutputBindings = <any> await actor.run(op);
+      await new Promise<void>(resolve => output.bindingsStream.on('end', () => resolve()));
```

Comment:
Not sure, but this may also work.
```suggestion
      await new Promise<void>(resolve => output.bindingsStream.on('end', resolve));
```